### DISCUSSION TITLE
Calendar with 360-day year

### DISCUSSION THREAD

**Discussion by sjh395** [Parcels-code/Parcels#1025] [Q&A] [answered]
I am trying to drive OceanParcels with palaeoclimate simulation data created with MITgcm.  In these types of simulations in which we are modelling the ocean of deep time (e.g. 65 Million Years Ago)- the standard calendar is not used. Instead we use an idealised 360 day year (12 x 30 day months)

I have input NetCDFs containing U, V and W, in which each NetCDF contains a time series dataset of 120 3-day means. The problem is in the creation of a numpy array of timestamps using the datetime64 data type - which uses the real calendar.

I was wondering if anyone has used 360-day ocean model data to drive OceanParcels? If I were to create a time:calendar attribute within the NetCDF and set it to 360_day and then set the time data accordingly, would OceanParcels parse this correctly?

Thanks in advance.
Stephen Hunter

**Comment by daanreijnders** [comment, marked as answer]
I use MITgcm in combination with xmitgcm to generate my netCDF files. I also use a 360-day calendar for convenience. Using xmitgcm's options for setting the calendar to 360 days (which relies on the `cftime` library), the data works fine with Parcels.

**Reply by daanreijnders** [reply]
See also #602 and #604

**Reply by sjh395** [reply]
Thank you Daan,
Thats good to know. I set the NetCDF Time attributes to 360_day and ensured my time vector and units were correct - this seems to have resolved my problem.

**Reply by daanreijnders** [reply]
You're welcome! I'm glad it helped
